spi/common/identity-trust-spi/src/main/java/org/eclipse/edc/identitytrust/model/VerifiableCredential.java

```diff
@@ -85,6 +88,10 @@ public String getName() {
         return name;
     }
 
+    public boolean isExpired() {
+        return issuanceDate.isAfter(Instant.now());
```

Comment:
Usage of `Clock`?

Reply:
we could even remove this method altogether. Some implementors may allow for a small epsilon (couple of minutes) to account for clock skew